projects/hslayers/services/layer-manager/layer-manager-folder.service.ts

```diff
@@ -199,21 +211,54 @@ export class HsLayerManagerFolderService {
       const zIndexes = folderChanged.layers.map((l) => l.layer.getZIndex());
       folderChanged.zIndex = Math.max(...zIndexes);
     }
-    return this.sortFoldersByZ(newState);
+    return this.sortFoldersByZ(newState, false);
   }
 
   /**
    * Sorts folders by z-index.
    */
   private sortFoldersByZ(
     state: Map<string, HsLayermanagerFolder>,
+    debounce = true,
   ): Map<string, HsLayermanagerFolder> {
```

Comment:
I would prefer if the `debounce` param will be and optional property inside of an (optional) options object. The ultimate goal is more readable function calls, instead of `sortFoldersByZ(state, false)`, it would be `sortFoldersByZ(state, {debounce: false})`. What do you think? We have discussed this some time ago I think.

Reply:
Sure, I just keep forgetting. Sorry 😄